source/common/event/dispatcher_impl.cc

```diff
@@ -280,12 +298,32 @@ MonotonicTime DispatcherImpl::approximateMonotonicTime() const {
   return approximate_monotonic_time_;
 }
 
+void DispatcherImpl::shutdown() {
+  ENVOY_LOG(debug, "Clearing dispatcher thread deletables in {}", __FUNCTION__);
```

Comment:
Is it helpful to know how many objects where deleted during shutdown?  If yes, consider returning the number of objects deleted from runThreadLocalDelete()